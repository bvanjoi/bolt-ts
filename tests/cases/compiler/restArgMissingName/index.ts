// From `github.com/microsoft/TypeScript/blob/v5.7.3/tests/cases/compiler/restArgMissingName.ts`, Apache-2.0 License

function sum (...) {} 
//~^ ERROR: Identifier expected.