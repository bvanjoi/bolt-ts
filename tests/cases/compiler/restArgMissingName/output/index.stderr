  × Identifier expected.
   ╭─[../../tests/cases/compiler/restArgMissingName/index.ts:3:18]
 3 │ )
   · ─
   ╰────
